core/src/read.jl

```diff
@@ -602,7 +657,7 @@ function parse_variables_and_conditions(compound_variable, condition)
     weight = Vector{Float64}[]
     look_ahead = Vector{Float64}[]
     greater_than = Vector{Float64}[]
-    condition_value = BitVector[]
+    condition_value = Vector{Bool}[]
```

Comment:
Why the change? A `Vector{Bool}` is actually 8 times larger in memory than `BitVector`. Performance should also be equal or better (largely) because of that.